### PR TITLE
data setup - fix imports

### PR DESCRIPTION
**Pull request by gmanchon**
`google-cloud-bigquery` >= 3 currently generates an error with the `db-dtypes` package during the training

### CODE REVIEW

**Comment by krokrob** [comment]
this closes #24

File: versioned/02/02_README_setup.md [hunk 1]
```diff
@@ -15,6 +15,12 @@ In this challenge, we will:
 
 Let's make sure that your setup is operational.
 
+You will need to install a few packages in order to play with GCP from your code:
```
Comment (krokrob) [inline versioned/02/02_README_setup.md:18]:
```suggestion
First, we need to install some useful Python packages to interact with GCP APIs from our project such as [Cloud Storage](https://cloud.google.com/storage/docs/apis) and [BigQuery](https://cloud.google.com/bigquery/docs/reference/rest):
```

**Review by krokrob — COMMENTED** [review]
thanks @gmanchon

**Comment by gmanchon** [comment]
nice addition, thanks @krokrob

**Pull request merged**